src/metaprob/sequence.clj

```diff
@@ -74,7 +74,7 @@
     (cond (seq? state) (count state)
           (vector? state) (count state)
           (map? state) (if (pair-as-map? state)
-                         (+ 1 (length (get state rest-marker)))
+                         (inc (length (get state rest-marker)))
```

Comment:
Same thing.  I don't know about `inc` so one might assume someone trying to read the code, who doesn't know clojure better than me, would stumble.  To me `inc` sounds like a side effect.  I would have to look it up.